data/sql/10-api-dos-db-role.sql

```diff
@@ -0,0 +1,33 @@
+--
+-- This script will create a role that the Capacity Status API will use to connect to
+-- the DoS database. The role will consist of the minmimum amount of priviledges required
+-- in order for the API to retrieve and update the capacity status information of a
+-- service.
+--
+SET default_transaction_read_only = off;
+
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+
+--
+-- Roles
+--
+CREATE ROLE uec_dos_api_cs_role;
+ALTER ROLE uec_dos_api_cs_role WITH NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB LOGIN NOREPLICATION NOBYPASSRLS PASSWORD 'uec_dos_api_cs_role_db_password';
```

Comment:
Naming is hard and I'm not claiming that my suggestion is any good but still want to share with you both @jope1 and @mabe13.

So, when deciding about naming we need to ask what's the context that is already known. This user is going to be created in the DoS database. We implemented that script but really it will be maintained by the DoS team and it will leave in their repository (we'll still keep the copy). Therefore, can we drop `uec_dos_` prefix?

In ProstgeSQL, role and user from a technical perspective are almost synonyms. Shall we remove `_role` as it doesn't add much value?

Now, we are left with just `api_cs` which is not informative enough. Then, what about `capacity_status_api` for the user name?

Also, can we change the password to `DOS_DB_PASSWORD_TO_REPLACE`? This will be replaced by the script on loading

Reply:
Thanks for sharing your thought process, I think this is sensible.

Reply:
agree 👍